### PR TITLE
Get jhu to pass pydocstyle

### PR DESCRIPTION
**Pull request by chinandrew**
### Description
Fix linting on jhu so pydocstyle passes.

### Changelog
Itemize code/test/documentation changes and files added/removed.
- Fix linting errors
- Enable pydocstyle to run in makefile so it gets run on CI

### Fixes 
- n/a

### CODE REVIEW

File: jhu/delphi_jhu/geo.py [hunk 1]
```diff
@@ -8,8 +8,7 @@
 
 def geo_map(df: pd.DataFrame, geo_res: str):
     """
-    Maps a DataFrame df, which contains data at the county resolution, and
-    aggregate it to the geographic resolution geo_res.
+    Map a DataFrame df the county resolution, and aggregate it to the geographic resolution geo_res.
```
Comment (krivard) [inline jhu/delphi_jhu/geo.py:11]: > Map a DataFrame df the county resolution,

Is this missing a word?

File: jhu/delphi_jhu/pull.py [hunk 2]
```diff
@@ -62,7 +64,7 @@ def sanity_check_data(df: pd.DataFrame) -> pd.DataFrame:
 
 
 def pull_jhu_data(base_url: str, metric: str, gmpr: GeoMapper) -> pd.DataFrame:
-    """Pulls the latest Johns Hopkins CSSE data, and conforms it into a dataset
+    """Pull the latest Johns Hopkins CSSE data, and conforms it into a dataset.
```
Comment (krivard) [inline jhu/delphi_jhu/pull.py:67]:
```suggestion
    """Pull the latest Johns Hopkins CSSE data, and conform it into a dataset.
```

**Review by krivard — CHANGES_REQUESTED** [review]

File: jhu/delphi_jhu/geo.py (hunk 1 above)
Comment (chinandrew) [inline jhu/delphi_jhu/geo.py:11]:
```suggestion
    Map and aggregate a DataFrame at the county resolution to the geographic resolution geo_res.
```

Comment (chinandrew) [reply jhu/delphi_jhu/geo.py]: oops thanks for catching, edited.

**Review by krivard — APPROVED** [review]
xlnt

**Pull request merged**